n-1 Satellite and Capsule upgrade support

jyejare opened the pull request:
The PR includes:

'n-1' capsule upgrade with satellite at 'n' version. 

Basically its like 'satellite' at latest downstream(zStream) and 'capsule' at released zStream.

jyejare commented:
After reviewing this PR, please review https://github.com/SatelliteQE/robottelo-ci/pull/571, which is dependent on this PR.

sghai commented on automation_tools/satellite6/upgrade/capsule.py:
```diff
@@ -20,7 +20,7 @@
 
 
-def satellite6_capsule_setup(sat_host, os_version):
+def satellite6_capsule_setup(sat_host, os_version, n_minus1=False):
```
can we update `n_minus1` to `capsule_update` or `update_capsule`?

sghai commented:
ACK pending a minor change.

jyejare commented:
@sghai Worked on your comment, and changed the parameter behavior ```i.e by default to True now``` as I changed parameter from n-1 cpasule to upgradable_capsule.

sghai commented:
ACK
